### DISCUSSION TITLE
posts/how-to-create-a-react-app-with-rust-and-wasm/?giscus=53c362dd20a97e53daafc7ecg7KswPstmI3DhFNbwPM8u%2BATWxI2ZdjtqQnuujROGILDAWCc4oS5tVLGImidkkXRkWYWxEIVMrJYKKHtoAtQHeMbCUO%2BwLeTMGcK%2FnV50IgT1xPcorqYKZ40Ny8%3D

### DISCUSSION THREAD

**Discussion by giscus** [tkat0/tkat0.github.io#11] [Blog Comments]
# posts/how-to-create-a-react-app-with-rust-and-wasm/?giscus=53c362dd20a97e53daafc7ecg7KswPstmI3DhFNbwPM8u%2BATWxI2ZdjtqQnuujROGILDAWCc4oS5tVLGImidkkXRkWYWxEIVMrJYKKHtoAtQHeMbCUO%2BwLeTMGcK%2FnV50IgT1xPcorqYKZ40Ny8%3D

In this article, I'll introduce followings through creating simple demo application.

https://www.tkat0.dev/posts/how-to-create-a-react-app-with-rust-and-wasm/

**Comment by empyrean987** [comment]
Could you add using fetch to this example?
https://rustwasm.github.io/wasm-bindgen/examples/fetch.html
